crates/consensus/beacon/src/engine/message.rs

```diff
@@ -0,0 +1,13 @@
+use reth_interfaces::consensus::ForkchoiceState;
+use reth_primitives::SealedBlock;
+use reth_rpc_types::engine::PayloadStatusEnum;
+use tokio::sync::oneshot;
+
+/// The message received by beacon consensus engine.
```

Comment:
nit: The terminology here is a bit confusing because of how engine API is named in the spec... Not sure how to make this clearer, although I think we should prefer using more descriptive language instead of past tense stuff (not sure what the correct grammar terms here are)

```suggestion
/// A message for the beacon engine from other components of the node.
```

Perhaps?